metricbeat/module/docker/_meta/config.yml

```diff
@@ -1,13 +1,5 @@
 - module: docker
-  metricsets: ["container", "cpu", "diskio", "healthcheck", "info", "memory", "network"]
   hosts: ["unix:///var/run/docker.sock"]
-  period: 10s
 
   # Replace dots in labels with `_`. Set to false to keep dots
   labels.dedot: true
```

Comment:
This is the default, so I guess it is not needed in `config.yml`.

Reply:
The reason I left this in is because we plan to change the default here in 7.0. So if it is in the config most people use, 7.0 will be less breaking for existing users.

Reply:
Oh ok, right then.